### PR TITLE
Fixing some warnings

### PR DESCRIPTION
**Pull request by PaulJonasJost**
Description of which warnings should be removed in commit messages

### CODE REVIEW

**Comment by PaulJonasJost** [comment]
Decreased from base: 570 to 68 warnings for now

**Review by vwiela — APPROVED** [review]
Looks good to me.

**Review by dilpath — APPROVED** [review]
:+1:

File: pypesto/store/read_from_hdf5.py [hunk 1]
```diff
@@ -121,10 +120,9 @@ def read(self, objective: ObjectiveBase = None) -> Problem:
         if objective is None:
             objective = Objective()
             # raise warning that objective is not loaded.
-            warnings.warn(
+            logger.debug(
```
Comment (dweindl) [inline pypesto/store/read_from_hdf5.py:123]: Using `warnings` here was intended to make it easier to disable those, see https://github.com/ICB-DCM/pyPESTO/pull/1253

**Pull request merged**